app/controllers/session_controller.rb

```diff
@@ -231,6 +232,49 @@ def create
     (user.active && user.email_confirmed?) ? login(user) : not_activated(user)
   end
 
+  def email_login_page
```

Comment:
Hi @fantasticfears , I'm refactoring this and reviewing this and was wondering why we need a page for this? Was there a reason why it wasn't handled like the admin login?